### ISSUE TITLE
[Package Request] - Add udica support for AL2022 and AL2

### ISSUE THREAD

**Issue by ayushman4** [amazonlinux/amazon-linux-2022#192]
**What package is missing from Amazon Linux 2022? Please describe and include package name.**
`Udica` ->  https://github.com/containers/udica. Udica is a tool that is used to generate custom SELinux policies for containers using udica templates in contianer-selinux package. Since container-selinux package is available using
```
# amazon-linux-extras enable selinux-ng
# yum install container-selinux
```
It makes sense that udica is installed so that AWS engineer and customers can use it to generate policies for containers as a defense in depth mechanism.

**Is this an update to existing package or new package request?**
new package request

**Is this package available in Amazon Linux 2?**
No

**Comment by karolpiatek-sidero** [comment]
udica is available in 2022.11.03 RC release 
and to fully use it we need stable container-selinux package,

https://github.com/amazonlinux/amazon-linux-2022/issues/218

**Comment by stewartsmith** [comment]
Since `udica` is shipped, will resolve this issue and we can track other bugs separately.
